### PR TITLE
DCOS-13109: Fix Load balanced checkbox for USER network

### PR DESCRIPTION
**Pull request by nLight**
This PR addresses the issue of the load balanced checkbox not removing VIP label from the app definition. It also keeps all non-VIP labels there as it is what we want to do according to the documentation.

### CODE REVIEW

**Comment by wavesoft** [comment]
Aside from DCOS-13206 works good! Nice ✨  💯

**Comment by wavesoft** [comment]
retest this please

**Comment by nLight** [comment]
run unit tests

File: src/js/constants/Networking.js [hunk 1]
```diff
@@ -5,7 +5,8 @@ const Networking = {
     HOST: 'HOST',
     USER: 'USER'
   },
-  L4LB_ADDRESS: '.marathon.l4lb.thisdcos.directory'
+  L4LB_ADDRESS: '.marathon.l4lb.thisdcos.directory',
+  VIP_LABEL_REGEX: /^VIP_[0-9]+$/
```
Comment (jfurrow) [inline src/js/constants/Networking.js]: 👍

**Comment by jfurrow** [comment]
Works well! Nice work!

File: plugins/services/src/js/utils/ServiceConfigUtil.js [hunk 2]
```diff
@@ -3,10 +3,16 @@ import Networking from '../../../../../src/js/constants/Networking';
 
 const ServiceConfigUtil = {
 
+  matchVIPLabel(str) {
+    return Networking.VIP_LABEL_REGEX.test(str);
+  },
+
+  findVIPLabel(labels = {}) {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/utils/ServiceConfigUtil.js]: Let's change this util to return a list of VIPs to acknowledge that fact that there can be more than an VIP label.

File: plugins/services/src/js/utils/ServiceConfigUtil.js [hunk 3]
```diff
@@ -3,10 +3,16 @@ import Networking from '../../../../../src/js/constants/Networking';
 
 const ServiceConfigUtil = {
 
+  matchVIPLabel(str) {
```
Comment (orlandohohmeier) [inline plugins/services/src/js/utils/ServiceConfigUtil.js]: Do we really need this util?

File: plugins/services/src/js/utils/ServiceConfigUtil.js [hunk 4]
```diff
@@ -3,10 +3,16 @@ import Networking from '../../../../../src/js/constants/Networking';
...
+
+  findVIPLabel(labels = {}) {
+    return Object.keys(labels).find(ServiceConfigUtil.matchVIPLabel);
+  },
+
   hasVIPLabel(labels = {}) {
-    return Object.keys(labels).find(function (key) {
-      return /^VIP_[0-9]+$/.test(key);
-    });
+    return !!ServiceConfigUtil.findVIPLabel(labels);
```
Comment (orlandohohmeier) [inline plugins/services/src/js/utils/ServiceConfigUtil.js]: Please don't cast boolean values as this can lead to unexpected results.

**Pull request merged**